e2e/tests/diagnosis.spec.ts

```diff
@@ -20,12 +19,13 @@ test('Create and revise a diagnosis.', async ({ page }) => {
   await bahmni.registerPatient();
 
   // replay
+  await bahmni.navigateToPatientDashboard();
   await bahmni.navigateToDiagnosis();
   await page.locator('#name-0').fill('Candidiasis (B37.9)');
   await page.getByText('Candidiasis (B37.9)').click();
   await page.locator('#order-0').getByRole('button', { name: /primary/i }).click();
   await page.locator('#certainty-0').getByRole('button', { name: /confirmed/i }).click();
-  await bahmni.saveOrder();
+  await bahmni.save();
```

Comment:
If this is about the order and cannot be called upon any other workflow process, then it'd be better keeping it as `saveOrder()`

Reply:
The method is called upon by other workflow processes.